channel/channel.go

```diff
@@ -27,6 +30,8 @@ type Channel struct {
 	// Longer term, we should have a more efficient and smart mechanism to store states https://github.com/statechannels/go-nitro/issues/106
 
 	latestSupportedStateTurnNum uint64 // largest uint64 value reserved for "no supported state"
+
+	StateTransitionValidator StateTransitionValidator // state transition validator
```

Comment:
Is it a mocked requireValidState App ? for injecting while testing?

Reply:
deprecated too